packages/react/src/lib/accessibility/FocusZone/AutoFocusZone.tsx

```diff
@@ -12,7 +12,7 @@ import getElementType from '../../getElementType'
 import callable from '../../callable'
 
 /** AutoFocusZone is used to focus inner element on mount. */
-export class AutoFocusZone extends React.Component<AutoFocusZoneProps> {
+export default class AutoFocusZone extends React.Component<AutoFocusZoneProps> {
```

Comment:
is this necessary or just an improvement?

Reply:
It's necessary to make working `defaultProps` parser because all our components has a default export. It's even covered by `isConformant()` test.